### PR TITLE
cli: add basic support for `constellation create` on OpenStack

### PR DESCRIPTION
**Pull request by malt3**
### Proposed change(s)

- image build for CSP OpenStack
- basic terraform template for OpenStack
- `constellation config generate` support for OpenStack
- `constellation create` support for OpenStack

### Additional info
- You need to set `CONSTELLATION_OPENSTACK_DEV=1` to test this feature
- You can use image `ref/feat-terraform-openstack/stream/debug/v2.6.0-pre.0.20230227132153-01d349e91e04` for testing
- Authentication requires either sourcing an openrc file or writing a valid `clouds.yaml` file.

### Checklist

- [x] Add labels (e.g., for changelog category)
- [ ] Link to Milestone

### CODE REVIEW

File: debugd/cmd/debugd/debugd.go [hunk 1]
```diff
@@ -81,6 +81,9 @@ func main() {
 		defer meta.Close()
 		fetcher = cloudprovider.New(meta)
 
+	// TODO: implement OpenStack
```
Comment (katexochen) [inline debugd/cmd/debugd/debugd.go:84]:
```suggestion
	// TODO(malt3): implement OpenStack
```

**Review by katexochen — CHANGES_REQUESTED** [review]

**Comment by derpsteb** [comment]
Didn't look at the code yet. But commands work as advertised :+1:

**Review by katexochen — APPROVED** [review]
lgtm :rocket:

**Comment by malt3** [comment]
@katexochen ptal at the fixup commits only

**Review by katexochen — APPROVED** [review]

**Pull request merged**